developer-docs-site/docs/nodes/full-node/update-fullnode-with-new-releases.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: "Update Fullnode With New Devnet Releases"
-slug: "update-fullnode-with-new-devnet-releases"
+title: "Update Fullnode With New Releases"
+slug: "update-fullnode-with-new-releases"
 sidebar_position: 11
 ---
 
 # Update Fullnode With New Releases
 
-When `devnet` is wiped and updated with newer versions, you will need to update your fullnode as well. If you do not, your fullnode will not continue to synchronize with the network. To update your fullnode, follow these steps:
+This document outlines the process for updating you fullnode. For fullnodes running in `devnet`, an additional data wipe step is required to allow the fullnode to continue syncrhonizing with the network.
```

Comment:
1. typo: `continue syncrhonizing` -> `continue synchronizing`
2. Also, I'd maybe say it's because we wipe devnet weekly instead, e.g.,:

```
For fullnodes running in `devnet`, an additional data wipe step is required as `devnet` is wiped on every release (usually around once a week).
```